src/librustc_error_codes/error_codes/E0708.md

```diff
@@ -0,0 +1,26 @@
+async` non-`move` closures with parameters are not currently supported
+
+Erroneous code example:
+
+```compile_fail,E0710
+#![feature(async_closure)]
+
+fn main() {
+    let add_one = async |num: u8| {
```

Comment:
You need to add a comment (at minimum `// error!`) on the failing line to help users identify the problem more visually.